.github/labeller.yml

```diff
@@ -0,0 +1,49 @@
+labels:
+  - name: "area-ciapp"
+    title: "^\\[?(?i)ciapp"
+
+  - name: "area-serverless"
+    title: "^\\[?(?i)serverless"
+
+  - name: "area-appsec"
+    title: "^\\[?(?i)appsec"
+
+  - name: "area-profiler"
+    title: "^\\[?(?i)profiler"
+
+  - name: "area-debugger"
+    title: "^\\[?(?i)debugger"
+
+  - name: "area-profiler"
+    allFilesIn: "profiler\/.*"
+
+  - name: "area-debugger"
+    allFilesIn: "debugger\/.*"
+
+  - name: "area:builds"
+    allFilesIn: "\\.github\/.*|tracer\/build\/.*|\\.azure-pipelines\/.*|gitlab-ci\\.yml"
+
+  - name: "area:vendors"
+    allFilesIn: "tracer\/src\/Datadog\\.Trace\/vendors\/.*"
+
+  - name: "area:docs"
+    allFilesIn: "docs\/.*|.*README.*|.*\\.md"
+
+  - name: "area:installers"
+    allFilesIn: "shared\/src\/msi-installer\/.*"
```

Comment:
Would files here also receive the `area:shared-components` label below? Ideally I would only want this label and not both